lib/coarse_op.cuh

```diff
@@ -8,7 +8,10 @@ namespace quda {
   // For coarsening un-preconditioned operators we use uni-directional
   // coarsening to reduce the set up code.  For debugging we can force
   // bi-directional coarsening.
-  static bool bidirectional_debug = false;
+  // ESW hack for now: need to add logic so bi-directional coarsening
+  // is triggered if ANY previous level is preconditioned, not just
+  // the current level.
+  static bool bidirectional_debug = true;
```

Comment:
This needs to be fixed, else it will double the cost of Wilson MG coarsening.

Reply:
Good catch, fixed.